composer.json

```diff
@@ -67,6 +68,7 @@
         "omnipay/2checkout": "~2.0",
         "omnipay/authorizenet": "~2.0",
         "omnipay/buckaroo": "~2.0",
+        "cardgate/omnipay-cardgate": "~2.0",
```

Comment:
@kayladnls we probably shouldn't add it to the require part, right? 3rd party code and all..